src/payment/strategies/digitalriver/digitalriver-payment-initialize-options.ts

```diff
@@ -1,5 +1,82 @@
 import { OptionsResponse } from './digitalriver';
 
+/**
+ * A set of options that are required to initialize the DigitalRiver payment method.
+ *
+ * When DigitalRiver is initialized, a widget will be inserted into the DOM. The widget has a list of payment options for the customer to choose from.
+ *
+ * ```html
+ * <!-- This is where the widget will be inserted -->
+ * <div id="container"></div>
+ * ```
+ *
+ * ```js
+ * service.initializePayment({
+ *   methodId: 'digitalriver',
+ *   digitalriver: {
+ *       containerId: 'digitalriver-component-field',
+ *       // Callback for submitting payment form that gets called when a buyer approves DR payment
+ *       onSubmitForm: () => {
+ *           // Example function
+ *           this.submitOrder(
+ *               {
+ *                   payment: {methodId: 'digitalriver',}
+ *               }
+ *           );
+ *       },
+ *       onError: (error) => {
+ *           console.log(error);
+ *       },
+ *   }
+ * });
+ * ```
+ *
+ * Additional options can be passed in to customize the components and register
+ * event callbacks.
+ *
+ * ```js
+ * service.initializePayment({
+ *   methodId: 'digitalriver',
+ *   digitalriver: {
+ *       containerId: 'digitalriver-component-field',
+ *       configuration: {
+ *           flow: 'checkout',
+ *           showSavePaymentAgreement: false,
+ *           showComplianceSection: true,
+ *           button: {
+ *               type: 'submitOrder',
+ *           },
+ *           usage: 'unscheduled',
+ *           showTermsOfSaleDisclosure: true,
+ *           paymentMethodConfiguration: {
+ *               disabledPaymentMethods: [
+ *                   'klarnaCredit',
+ *                   'payPal',
+ *                   'payPalCredit',
+ *                   'payPalBilling',
+ *               ],
+ *               classes: {
+ *                   base: 'form-input optimizedCheckout-form-input'
```

Comment:
Do we need to say what these classes do?

Reply:
Hey @animesh1987, AFAIK yes, because we are using these classes to control styles on BC side

Reply:
Hey @animesh1987  Is this PR approved?? or Do you have any other question??

Reply:
My question was around, does it make sense to add a comment as to what those classes mean. Since they are important/required from what @AlanMunozN said

Reply:
sure, I added a comment